examples/walkthrough/main.go

```diff
@@ -129,6 +140,114 @@ func CreatePublicSharedLink(ctx context.Context, accessGrant, bucketName, object
 	return url, nil
 }
 
+func MultipartUpload(ctx context.Context, accessGrant, bucketName, objectKey string, contents1, contents2 []byte) error {
```

Comment:
```suggestion
// MultipartUpload demonstrates how to upload object in multiple parts.
func MultipartUpload(ctx context.Context, accessGrant, bucketName, objectKey string, first, second []byte) error {
```

The change from `contents1` -> to `first`, because when reading code the suffix numbers can get confusing. Especially when they are interleaved in places. It's not a big problem in this case, but we like to keep this consistent. (Also the relevant error messages need updating as well.)

Using `firstPart` or `firstContent` would be also fine.